packages/xrpl/test/integration/transactions/permissionedDomain.test.ts

```diff
@@ -0,0 +1,90 @@
+import { stringToHex } from '@xrplf/isomorphic/utils'
+import { assert } from 'chai'
+
+import {
+  LedgerEntryRequest,
+  PermissionedDomainDelete,
+  PermissionedDomainSet,
+  AuthorizeCredential,
+} from '../../../src'
+import PermissionedDomain from '../../../src/models/ledger/PermissionedDomain'
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('PermissionedDomainSet', function () {
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'Lifecycle of PermissionedDomain ledger object',
+    async () => {
+      const sampleCredential: AuthorizeCredential = {
+        Credential: {
+          CredentialType: stringToHex('Passport'),
+          Issuer: testContext.wallet.classicAddress,
+        },
+      }
+
+      // Step-1: Test the PermissionedDomainSet transaction
+      const tx_pd_set: PermissionedDomainSet = {
```

Comment:
Use `camelCase`, not `snakeCase` in JS

Reply:
[9154151](https://github.com/XRPLF/xrpl.js/pull/2874/commits/91541510cb43525583d0c32c3acec258831fd230)